scheduler/google/cloud/scheduler_v1/proto/cloudscheduler_pb2_grpc.py

```diff
@@ -0,0 +1,196 @@
+            response_deserializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_cloudscheduler__pb2.ListJobsResponse.FromString,
+        )
+        self.GetJob = channel.unary_unary(
+            "/google.cloud.scheduler.v1.CloudScheduler/GetJob",
+            request_serializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_cloudscheduler__pb2.GetJobRequest.SerializeToString,
+            response_deserializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_job__pb2.Job.FromString,
+        )
+        self.CreateJob = channel.unary_unary(
+            "/google.cloud.scheduler.v1.CloudScheduler/CreateJob",
+            request_serializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_cloudscheduler__pb2.CreateJobRequest.SerializeToString,
+            response_deserializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_job__pb2.Job.FromString,
+        )
+        self.UpdateJob = channel.unary_unary(
+            "/google.cloud.scheduler.v1.CloudScheduler/UpdateJob",
+            request_serializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_cloudscheduler__pb2.UpdateJobRequest.SerializeToString,
+            response_deserializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_job__pb2.Job.FromString,
+        )
+        self.DeleteJob = channel.unary_unary(
+            "/google.cloud.scheduler.v1.CloudScheduler/DeleteJob",
+            request_serializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_cloudscheduler__pb2.DeleteJobRequest.SerializeToString,
+            response_deserializer=google_dot_protobuf_dot_empty__pb2.Empty.FromString,
+        )
+        self.PauseJob = channel.unary_unary(
+            "/google.cloud.scheduler.v1.CloudScheduler/PauseJob",
+            request_serializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_cloudscheduler__pb2.PauseJobRequest.SerializeToString,
+            response_deserializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_job__pb2.Job.FromString,
+        )
+        self.ResumeJob = channel.unary_unary(
+            "/google.cloud.scheduler.v1.CloudScheduler/ResumeJob",
+            request_serializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_cloudscheduler__pb2.ResumeJobRequest.SerializeToString,
+            response_deserializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_job__pb2.Job.FromString,
+        )
+        self.RunJob = channel.unary_unary(
+            "/google.cloud.scheduler.v1.CloudScheduler/RunJob",
+            request_serializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_cloudscheduler__pb2.RunJobRequest.SerializeToString,
+            response_deserializer=google_dot_cloud_dot_scheduler__v1_dot_proto_dot_job__pb2.Job.FromString,
+        )
+
+
+class CloudSchedulerServicer(object):
+    """The Cloud Scheduler API allows external entities to reliably
+  schedule asynchronous jobs.
+  """
+
+    def ListJobs(self, request, context):
+        """Lists jobs.
+    """
+        context.set_code(grpc.StatusCode.UNIMPLEMENTED)
```

Comment:
I'm assuming it's not normal for all the methods to have this?

Reply:
I have seen this on other clients. We usually use the `cloudscheduler_pb2` import though. @lukesneeringer ?

Reply:
I have no idea what the gRPC stubs do. _shrug_